lib/strategies/webapp-strategy.js

```diff
@@ -207,6 +208,36 @@ function handleAuthorization(req, options, strategy) {
 		options.successRedirect = req.url;
 	}
 
```

Comment:
Can you please add some comments that explains the code flow (e.g. "if no refresh method defined by user or no refresh token resolved, or refresh token validation failed on server, then proceed with authorization flow").